docs/how-to/create-a-router/configure-traffic-rules.md

```diff
@@ -51,10 +51,12 @@ From the above rules, both rule 1 and 2 are not overlapping because there's at l
 
 ### Conditions
 
-Each rule should have at least one condition configured on it. If there are multiple conditions configured on the same rule, then this rule will be triggered only if each and every condition is satisfied. Rule condition can be defined on either request header or request payload (assuming payload is a valid JSON object). For each condition you should specify:
+Each rule should have at least one condition configured on it. If there are multiple conditions configured on the same rule, then this rule will be triggered only if each and every condition is satisfied. 
 
-* **Condition source**: either `Header` or `Payload`
-* **Condition key**: if condition's source is `Header` – then the name of a request Header (example: `X-Session-ID`), or else, if condition's source is `Payload` – a valid JSON path of the property from the request's JSON payload (example: `service_type.id` or `users.0.name`)
+Rule condition can be defined on either request header or request payload (assuming payload is a valid JSON object). For UPI routers, it would be header or [prediction context](https://github.com/caraml-dev/universal-prediction-interface/blob/main/proto/caraml/upi/v1/upi.proto) (name of the variable and value). For each condition, you should specify:
+
+* **Condition source**: either `Header` or `Payload` for HTTP router. `Header` or `Prediction Context` for UPI router.
+* **Condition key**: if condition's source is `Header` – then the name of a request Header (example: `X-Session-ID`), or else, if condition's source is `Payload` – a valid JSON path of the property from the request's JSON payload (example: `service_type.id` or `users.0.name`). If condition source is `Prediction Context` - the name of the variable within Prediction Context proto. 
```

Comment:
Nit: `within Prediction Context proto` -> ` within the Prediction Context proto`